Libraries/Azure.psm1

```diff
@@ -2332,8 +2332,20 @@ Function Set-SRIOVinAzureVMs {
             if ( $VMPropertiesChanged ) {
                 # Start the VM..
                 Write-LogInfo "Starting VM $($VMName)..."
-                $null = Start-AzVM -ResourceGroup $ResourceGroup -Name $VMName
-
+                $null = Start-AzVM -ResourceGroupName $ResourceGroup -Name $VMName -NoWait
+                if ($? -eq "True") {
+                    Write-Loginfo "Start-AzVM command executes successfully."
+                } else {
+                    Write-LogErr "Start-AzVM command executes failed."
+                }
+                $vm = Get-AzVM -ResourceGroupName $ResourceGroup -Name $VMName -Status
+                $MaxAttempts = 10
+                while (($vm.Statuses[-1].Code -ne "PowerState/running") -and ($MaxAttempts -gt 0)) {
```

Comment:
This design does not work with big size VM, HB, HC, HBv2 and Mv2. Maybe there will be a new whitelist for those VMs and extend the max time to 20?